content/it/community/code-of-conduct.md

```diff
@@ -8,17 +8,18 @@ css: /css/community.css
 <div class="community_main">
 <h1>Codice di condotta della comunità di Kubernetes</h1>
 
-Kubernetes segue il 
+Kubernetes segue il
 <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">codice di condotta CNCF</a>.
-Il testo del CNC CoC è replicato di seguito a partire dal 
-<a href="https://github.com/cncf/foundation/blob/0ce4694e5103c0c24ca90c189da81e5408a46632/code-of-conduct.md">commit 0ce4694</a>.
-Se noti che questo non è aggiornato, ti preghiamo di far presente questo problema.
-<a href="https://github.com/kubernetes/website/issues/new">file an issue</a>.
+Il testo del codice di condotta CNCF è replicato di seguito a partire dal
+<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+Se noti che questo non è aggiornato, ti preghiamo
+<a href="https://github.com/kubernetes/website/issues/new">inviare un rapporto</a>
```

Comment:
rapporto --> issue
I think that translating terms that are so broadly used might create more confusion than benefits, wdyt?